core/hw/naomi/naomi_cart.cpp

```diff
@@ -204,13 +204,14 @@ bool naomi_cart_LoadRom(char* file)
 bool naomi_cart_SelectFile(void* handle)
 {
 	cfgLoadStr("config", "image", SelectedFile, "null");
-	
+
 #if HOST_OS == OS_WINDOWS
-	if (strcmp(SelectedFile, "null") == 0) {
+	if (SelectedFile == "null")
+    {
 		OPENFILENAME ofn = { 0 };
 		ofn.lStructSize = sizeof(OPENFILENAME);
 		ofn.hInstance = (HINSTANCE)GetModuleHandle(0);
-		ofn.lpstrFile = SelectedFile;
+		ofn.lpstrFile = &SelectedFile[0];
```

Comment:
why not `SelectedFile.c_str()`?

Reply:
lpstrFile is long pointer string, c_str() return constant char pointer.

Reply:
`ofn.lpstrFile` is also used as output here? Is `SelectedFile` big enough?